README.md

```diff
@@ -2,18 +2,24 @@
 ![Testspace Action](https://github.com/testspace-com/setup-testspace/workflows/Testspace%20Action/badge.svg)
 
 # Testspace client Setup JavaScript Action
-A GitHub Action used to install and configure the Testspace client used for publishing test content to [Testspace.com](https://testspace.com). 
+A GitHub Action is used to install and configure the Testspace client used for publishing test content to [Testspace.com](https://testspace.com). 
 
 ## Usage
 Setting up the Testspace client:
 
 ```yaml
-uses: testspace-com/setup-testspace@v1
+uses: testspace-com/setup-testspace@v2
```

Comment:
I don't think v2 is needed as there are no interface or functionality changes! Besides that we don't have much usage to justify so many versions.